Source/JavaScriptCore/dfg/DFGClobberize.h

```diff
@@ -270,13 +270,9 @@ void clobberize(Graph& graph, Node* node, const ReadFunctor& read, const WriteFu
         def(PureValue(node));
         return;
 
-    case GlobalIsNaN: {
-        if (node->child1().useKind() == DoubleRepUse)
-            def(PureValue(node));
-        else
-            clobberTop();
+    case GlobalIsNaN:
+        clobberTop();
```

Comment:
Add an assertion for `node->child1().useKind() == UntypedUse`?

Reply:
Added.